Makefile

```diff
@@ -36,10 +36,10 @@ fabric-docker-images:
 
 .PHONY: weaver-docker-images
 weaver-docker-images:
-	docker pull ghcr.io/hyperledger-labs/weaver-fabric-driver:1.2.1
-	docker image tag ghcr.io/hyperledger-labs/weaver-fabric-driver:1.2.1 hyperledger-labs/weaver-fabric-driver:latest
-	docker pull ghcr.io/hyperledger-labs/weaver-relay-server:1.2.1
-	docker image tag ghcr.io/hyperledger-labs/weaver-relay-server:1.2.1 hyperledger-labs/weaver-relay-server:latest
+	docker pull ghcr.io/hyperledger-labs/weaver-fabric-driver:1.4.2
+	docker image tag ghcr.io/hyperledger-labs/weaver-fabric-driver:1.4.2 hyperledger-labs/weaver-fabric-driver:latest
+	docker pull ghcr.io/hyperledger-labs/weaver-relay-server:1.4.2
+	docker image tag ghcr.io/hyperledger-labs/weaver-relay-server:1.4.2 hyperledger-labs/weaver-relay-server:latest
```

Comment:
Can you please use `$(WEAVER_VERSION)` and set it at the top of the file next to the FABRIC_VERSION

Reply:
fixed